packages/plugin-core/src/components/views/NoteGraphViewFactory.ts

```diff
@@ -141,6 +153,35 @@ export class NoteGraphPanelFactory {
             }
             break;
           }
+          case GraphViewMessageEnum.onGraphThemeChange: {
+            this.defaultGraphTheme = msg.data.graphTheme;
```

Comment:
let's add a telemetry marker on this - whenever the user changes their theme + what theme they changed to.